lib/src/signed_video_tlv.c

```diff
@@ -132,6 +132,25 @@ static const sv_tlv_tuple_t tlv_tuples[] = {
     {NUMBER_OF_TLV_TAGS, 0, NULL, NULL, true},
 };
 
+/*
+ * This is an array that contains only optional tags (not |is_always_present|).
+ * It is necessary to generate golden SEIs and when the golden SEI principle is not used.
```

Comment:
You can leave out this line. If we see `signed_video_tlv` as a file that simply handles the TLV structure without asking any questions, it is of no interest to know how or when it is used. This comment belongs in the `sign` file.